toptal/picasso
# chore: [FX-2178] Remove colored container borders

[FX-2178](https://toptal-core.atlassian.net/browse/FX-2178)

### Description

- Colored containers should not have borders
- Only white container can have a border

### How to test

- Visit [Picasso Container Component](https://picasso.toptal.net/fx-2178-container-remove-colored-borders/?path=/story/layout-container--container). **Bordered variant** should have only one example. Click on `edit code`, and add `variant='red'` (or any other available color) to the `Container` component, it should not add borders to the element and typescript should also complain: ![Screenshot from 2021-12-06 11-56-48](https://user-images.githubusercontent.com/72510037/144808613-44ce617a-2722-4e15-a57d-24176e326b7c.png) ![Screenshot from 2021-12-06 11-58-22](https://user-images.githubusercontent.com/72510037/144808808-57906a0b-5a9d-4e7c-acbc-4c207422e6a1.png)

### Screenshots

| Before.                                 | After.                                  |
| --------------------------------------- | --------------------------------------- |
| ![Screenshot from 2021-12-06 11-31-22](https://user-images.githubusercontent.com/72510037/144805486-97d3102f-6fe4-4e04-8bd5-e9df2e377e0d.png) | ![Screenshot from 2021-12-06 11-30-46](https://user-images.githubusercontent.com/72510037/144805528-8932c0fe-e83c-4d02-8375-ab09a22142e5.png) |

### Review



<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and
1. You have no extra requests.
2. You have optional requests.
   1. Add `nit: ` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because
1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:
1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>
